### PR TITLE
Issue/details page pt2

### PR DESCRIPTION
**Pull request by ntsummers1**
# Title

Issue/details page pt2

# Summary

This updates the styling for the display page. Makes it look snazzy!

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How To Test

# Checklist:

- [x] My code follows the style guidelines of this project and has no linting errors
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, including any applicable ADRs
- [x] My changes generate no new warnings
- [x] I have added tests that fail without these changes
- [x] New and existing tests (unit, integration, accessibility) pass locally
- [ ] Documentation updated
- [ ] If there are security concerns they are addressed or ticketed after being discussed

### CODE REVIEW

File: src/app/components/cards/BannerCard.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+              subtitle={"Average cost per year"}
+              highlight={maskCurrency(college.avgCost)}
+              icon="local_offer"
+            />
+          </div>
+        </CardBody>
+      </div>
+    </Card>
+  );
+};
```
Comment (gmrabian) [inline src/app/components/cards/BannerCard.tsx:60]: do we still want to have an apply button from this page?

**Review by gmrabian — COMMENTED** [review]
super appreciate you doing this!
do we want an apply button on the details page?

File: src/app/components/cards/BannerCard.tsx (hunk 1 above)
Comment (ntsummers1) [reply src/app/components/cards/BannerCard.tsx:60]: Gooooood catch. Added that back in!

**Comment by TheDanMiller** [comment]
> super appreciate you doing this! do we want an apply button on the details page?

We do want an apply button

**Review by gmrabian — APPROVED** [review]

**Pull request merged**